README.md

```diff
@@ -154,6 +156,10 @@ Usage of ./sish:
         Whether or not to redirect the root domain (default true)
   -sish.redirectrootlocation string
         Where to redirect the root domain to (default "https://github.com/antoniomika/sish")
+  -sish.serverlogformat string
```

Comment:
This help output is generated from the data stored as `flag` comments. Would you mind updated those to reflect these and then regenerating this help output from a `go run . -h`?

Reply:
Gotcha, no problem